virttest/env_process.py

```diff
@@ -1040,8 +1040,8 @@ def postprocess(test, params, env):
                 vm.name)
             vm.destroy()
 
-    # Terminate the tcpdump thread
-    env.stop_tcpdump()
+    # Terminate the ip sniffer thread
+    env.stop_ip_sniffer()
```

Comment:
It'd be nice to separate the commits. First move the code around to use `start_sniffer` etc. and then add the next sniffer.